### PR TITLE
#349 Remove checkServerVersion

### PR DESCRIPTION
**Pull request by tw-ayush**
Remove checkServerVersion #349

https://github.com/matterpoll/matterpoll/issues/349

### CODE REVIEW

File: server/plugin/plugin_test.go [hunk 1]
```diff
@@ -71,7 +71,6 @@ func TestPluginOnActivate(t *testing.T) {
 				m := semver.MustParse(manifest.MinServerVersion)
```
Comment (kaakaa) [inline server/plugin/plugin_test.go:71]: This test case ("greater minor version than minimumServerVersion") seems to be same as next test case. Would you remove this case?

File: server/plugin/plugin_test.go [hunk 2]
```diff
@@ -88,8 +87,6 @@ func TestPluginOnActivate(t *testing.T) {
 		},
 		"same version as minimumServerVersion": {
```
Comment (kaakaa) [inline server/plugin/plugin_test.go:88]: This test case name should be changed.

```suggestion
		"all fine": {
```

**Review by kaakaa — COMMENTED** [review]
Thanks for your contribution @tw-ayush 
I left a few comments about test cases.

File: server/plugin/plugin_test.go (hunk 1 above)
Comment (tw-ayush) [reply server/plugin/plugin_test.go]: Done

File: server/plugin/plugin_test.go (hunk 2 above)
Comment (tw-ayush) [reply server/plugin/plugin_test.go]: Done

**Comment by tw-ayush** [comment]
also can you please make the issue hacktoberfest eligible?

**Comment by kaakaa** [comment]
Thanks @tw-ayush! And since Matterpoll repository has "hacktoberfest" topic, this PR should be eligible to hacktoberfest.

**Comment by kaakaa** [comment]
Checks didn't run successfully, so I'll try to close and reopen this.

**Comment by tw-ayush** [comment]
@hanzei are there any changes i need to do make this merge-able?

**Review by hanzei — APPROVED** [review]
LGTM, but I leave the final say to @kaakaa

**Review by kaakaa — APPROVED** [review]
@tw-ayush 
LGTM 👍 
we'll merge this PR later due to release process, but hacktoberfest will count this contribution by approval.

**Comment by hanzei** [comment]
Thanks for the contribution @tw-ayush :+1:

**Pull request merged**